src/pages/docs/infrastructure/workers/autoscaling-workers/cluster-configuration.md

```diff
@@ -10,25 +10,30 @@ navOrder: 20
 The Kubernetes worker has been proven to be effective on a variety of installations.
 But some configurations are more complex than others!
 
-There's three factors which affect the effort the likelihood of success:
+There's three factors which affect the likelihood of success:
 1. Kubernetes distribution (eg AKS, EKS, GKE ...)
 2. Storage provider type (i.e. the filesystem shared between worker and pods)
 3. The Operating System of the Kubernetes nodes
 
 When trying to determine the best combination of these for your situation, it may be simplest to start small and iterate.
 
-| Distribution  | Storage Solution: | Approach                                               |
-|:-------------:|:-----------------:|--------------------------------------------------------|
-|      AKS      |        NFS        | No additional configuration required                   |
-|               |    Azure Files    | No additional configuration required                   |
-|      GKE      |        NFS        | No additional configuration required                   |
-|      EKS      |        NFS        | No additional configuration required                   |
-|               |        EFS        | Requires Octopus Server 2024.3+                        |
-|     RKE2      |     Longhorn      | Requires pre-configured storage - email [support@octopus.com](mailto:support@octopus.com) |
-|   OpenShift   |        NFS        | Requires specific configuration - email [support@octopus.com](mailto:support@octopus.com) |
-
-While the above table defines known good configurations, there are many other configurations which are likely to 
+The following table defines known good configurations, though there are many other configurations which are likely to
 produce a valid system.
+
+| Distribution | Storage Solution: | Approach                                               |
+|:------------:|:-----------------:|--------------------------------------------------------|
+|   Minikube   | NFS | No additional configuration required, recommended for PoC only |
```

Comment:
Should we add K3s?

Also, should we say that these distributions are for PoCs only? Customers might use them for Edge, it's a legitimate use case.